1-src/1-api/2-auth/auth-utilities.mts

```diff
@@ -1,34 +1,52 @@
 import dotenv from 'dotenv';
 import { NextFunction, Request, Response } from 'express';
-import JWT_PKG, { JwtPayload } from 'jsonwebtoken';
+import jwtPackage, { JwtPayload } from 'jsonwebtoken';
 import { createHash } from 'node:crypto';
 import { RoleEnum } from '../../0-assets/field-sync/input-config-sync/profile-field-config.mjs';
 import USER from '../../2-services/1-models/userModel.mjs';
 import { DB_SELECT_USER, DB_SELECT_USER_PROFILE } from '../../2-services/2-database/queries/user-queries.mjs';
 import * as log from '../../2-services/log.mjs';
 import { JwtData } from './auth-types.mjs';
 import { LoginResponseBody } from '../../0-assets/field-sync/api-type-sync/auth-types.mjs';
+import { GetSecretValueCommand, GetSecretValueResponse, SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
 dotenv.config(); 
-const {sign, verify, decode} = JWT_PKG;
 
 /********************
    Create secret key
 ********************/
-var APP_SECRET_KEY;
+let APP_SECRET_KEY:string | undefined;
 
-export const generateSecretKey = () => {
-   const time = new Date().getTime();
-   APP_SECRET_KEY = createHash('sha256').update(time + process.env.SECRET_KEY).digest('base64');
```

Comment:
Since w're not creating hash anymore, is it assumed the value in SecretsMaangerClient is a hash?

Reply:
Just saw your planning to use `KMS's GenerateDataKey`